roles/tempo/templates/config.yaml

```diff
@@ -0,0 +1,44 @@
+# EXPERIMENTAL!
+search_enabled: true
+
+server:
+  http_listen_address: {{ monitoring_host_internal_addr }}
+  http_listen_port: 3300
+  grpc_listen_address: 127.0.0.1
+  grpc_listen_port: 9095               # default
+
+distributor:
+  log_received_traces: true
+  receivers:                           # this configuration will listen on all ports and protocols that tempo is capable of.
+    jaeger/withendpoint:               # the receives all come from the OpenTelemetry collector.  more configuration information can
+      protocols:                       # be found there: https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver
+        thrift_http:
+          endpoint: {{ monitoring_host_internal_addr }}:14268
+
+ingester:
+  trace_idle_period: 10s               # the length of time after a trace has not received spans to consider it complete and flush it
```

Comment:
This should be longer; it's quite common for the crawler to take up to 3 minutes to scrape. Perhaps it could be 5 minutes?

Reply:
https://github.com/ipfs-search/ipfs-search-deployment/commit/5b682ab62418c1aeb72fabb5002035b36527db1b